src/transform/deadCode.ml

```diff
@@ -0,0 +1,217 @@
+open Batteries
+open GoblintCil
+open GobConfig
+open MyCFG
+
+let dpf fmt = Pretty.gprintf (fun doc -> print_endline @@ Pretty.sprint ~width:Int.max_num doc) fmt
```

Comment:
All of the custom debug printing should be removed or replaced by tracing.

Reply:
Missed that, thanks.